portlets/src/main/webapp/WEB-INF/conf/challenges/portal/portal/global/navigation.xml

```diff
@@ -25,10 +25,10 @@ Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 
     <page-nodes>
         <node>
-            <name>challenges</name>
-            <label>#{portal.global.challenges}</label>
+            <name>engagement</name>
+            <label>#{portal.global.engagement}</label>
             <visibility>SYSTEM</visibility>
-            <page-reference>portal::global::challenges</page-reference>
+            <page-reference>portal::global::engagement</page-reference>
```

Comment:
IMO we should keep both navigations **/challenges**  and **/engagement** for backward compatibility until we can change the url for all apps (activities, notifs ..), so at this stage we can use the 2 navigations for the same page  "**engagement**"

Reply:
okay, so should I create another <node> or add just another page refrence

Reply:
another node referring to the same page.

Reply:
done, you can find the changes in the following commit, thank you @AzmiTouil